src/app/util/util.cpp

```diff
@@ -15,12 +15,14 @@
  *    limitations under the License.
  */
 
-#include "app/util/common.h"
+#include "util.h"
```

Comment:
since a lot of platform code might have some form of util.h somewhere in their SDK, should we use the complete path instead `/app/util/util.h` ? Also it preserve the original include form